frontend/src/models/AutoScheduleFrequency.ts

```diff
@@ -30,4 +30,14 @@ export const allDays = [
     DaysOfWeek.Sunday,
 ]
 
+export const allDaysStartingSunday = [
+    DaysOfWeek.Sunday,
+    DaysOfWeek.Monday,
+    DaysOfWeek.Tuesday,
+    DaysOfWeek.Wednesday,
+    DaysOfWeek.Thursday,
+    DaysOfWeek.Friday,
+    DaysOfWeek.Saturday,
+]
```

Comment:
Do we need both allDays and allDaysStartingSunday? 

One possibility is this, or something similar using index of sunday
allDaysStartingSunday = allDays.slice(6).concat(allDays.slice(0, 6))